bedrock/releasenotes/tests/test_base.py

```diff
@@ -51,6 +51,7 @@ def test_get_release_or_404(self, Q, get_object_or_404):
         get_object_or_404.assert_called_with(
             Release, Q.return_value, version='version')
         Q.assert_called_with(product='product')
+        Q.__or__.assert_not_called()
```

Comment:
I don't think `assert_not_called` is a thing. Can't find it in the [Mock docks](http://www.voidspace.org.uk/python/mock/mock.html) anyway. I think you have to do `ok_(not Q.__or__.called)`. The `assert_not_called` thing is passing because it's just calling a mock function.